packages/authentication/src/definitions.ts

```diff
@@ -106,6 +107,11 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.1.0
    */
   getIdToken(options?: GetIdTokenOptions): Promise<GetIdTokenResult>;
+  /**
+   * Fetches the Firebase Auth ID Token and other helper properties for getting different data associated with the token as well as all the decoded payload claims for the currently signed-in user.
+   * 
```

Comment:
```suggestion
   * 
   * @since 6.4.0
```

Add some version information.